style.css

```diff
@@ -400,3 +400,214 @@ form input,
   background-color: rgba(255, 255, 255, 0.3);
   border: rgba(255, 255, 255, 1);
 }
+
+@media screen and (min-width: 992px) {
+  #navbar {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 20px 0;
+    background-color: #fff;
+  }
+
+  .links {
+    display: block;
+    background: none;
+    backdrop-filter: blur(0);
+    padding: 0;
+    mix-blend-mode: normal;
+    overflow: visible;
+    width: 100%;
+    height: 0;
+  }
+
+  #hamBtn {
+    display: none;
+  }
+
+  #hamBtnClose {
+    display: none;
+  }
+
+  .links li {
+    margin: 0 30px;
+  }
+
+  #toolbar {
+    margin-left: 155px;
+  }
+
+  .nav__links {
+    margin-top: 20px;
+    padding-right: 155px;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .nav__links a {
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  .background {
+    background-image: url(imgs/mobile-background.svg);
+    background-color: #fff;
+    background-repeat: no-repeat;
+    background-size: 90%;
+    height: 100%;
+    border-radius: 0 0 0 110px;
+  }
+
```

Comment:
- [ ] It would be better to add padding to the bottom of heading section to improve your UI design and also to match the Figma design.
- Your design:
<img width="952" alt="image" src="https://user-images.githubusercontent.com/77190203/155863252-7be40293-d7f8-4c34-9e94-9c9a69499fbd.png">
 
Figma design: